modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementSuccessTest.java

```diff
@@ -258,7 +249,44 @@ public void testGetOrganization() {
                 .body("id", equalTo(organizationID));
     }
 
-    @Test(dependsOnMethods = "testGetOrganization")
+    @DataProvider(name = "filterQueries")
+    public Object[][] createFilterQueries() {
```

Comment:
```suggestion
    public Object[][] dataProviderForFilterOrganizations() {
```

Reply:
This is now updated to `dataProviderForFilterOrganizations`